.github/workflows/ci-docs.yml

```
@@ -0,0 +1,28 @@
+      - uses: actions/checkout@v4
+
+      - name: Setup Bun
+        uses: oven-sh/setup-bun@v2
+        with:
+          bun-version: latest
+
+      - name: Install deps (minimal)
+        run: bun install --frozen-lockfile
+        shell: cmd
+
+      - name: Docs: link check
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 22: mapping values are not allowed in this context

(syntax-check)